cypress/integration/single-beacon-owner/i-can-remove-a-use.spec.ts

```diff
@@ -1,60 +1,93 @@
 import { Purpose } from "../../../src/lib/registration/types";
 import { givenIHaveEnteredMyBeaconDetails } from "../common/i-can-enter-beacon-information.spec";
+import {
+  givenIHaveEnteredMyAviationUse,
+  iCanSeeMyAviationUse,
+} from "../common/i-can-enter-use-information/aviation.spec";
 import { andIHaveAnotherUse } from "../common/i-can-enter-use-information/generic.spec";
 import {
   givenIHaveEnteredMyLandUse,
+  iCanEditMyAdditionalLandUseMoreDetails,
   iCanSeeMyLandUse,
 } from "../common/i-can-enter-use-information/land.spec";
 import {
   givenIHaveEnteredMyMaritimeUse,
   iCanSeeMyMaritimeUse,
 } from "../common/i-can-enter-use-information/maritime.spec";
-import { givenIHaveClickedTheButtonContaining } from "../common/selectors-and-assertions.spec";
+import {
+  givenIHaveClickedTheButtonContaining,
+  whenIClickBack,
+} from "../common/selectors-and-assertions.spec";
 
 describe("As a beacon owner with several uses", () => {
   it("I can safely remove a use from my draft registration", () => {
-    givenIHaveTwoUses();
+    givenIHaveThreeUses();
     andIGoToDeleteMyMainUse();
-    iAmPromptedToConfirmDeletion();
+    iAmPromptedToConfirmDeletionOfMyMainUse();
 
     givenIHaveClickedTheButtonContaining("Cancel");
-    iCanSeeMyTwoUses();
+    iCanSeeMyThreeUses();
     andIGoToDeleteMySecondUse();
-    // TODO: iAmPromptedToConfirmDeletion();
-    // TODO: givenIHaveClickedTheButtonContaining("Yes");
-    // TODO: iCanConfirmMyDeletionChoice;
-    // TODO: iCanSeeMyUsesWithoutTheDeletedUse
+    iAmPromptedToConfirmDeletionOfMySecondUse();
+
+    givenIHaveClickedTheButtonContaining("Yes");
+    iCanSeeMyMainUse();
+    iCannotSeeWhatWasMySecondUseBecauseItIsDeleted();
+    myThirdUseIsNowMySecondUse();
+
+    whenIClickBack();
+    iAmEditingWhatIsNowMySecondUse();
   });
 });
 
-const givenIHaveTwoUses = () => {
+const iCanSeeMyMainUse = () => iCanSeeMyLandUse();
+
+const myThirdUseIsNowMySecondUse = () =>
+  cy.get("h2").contains(/third use/i && /aviation/i && /commercial/i);
```

Comment:
I'm just having a look at it myself locally, but should this be `second use` rather than `third`? 🤔  
```suggestion
  cy.get("h2").contains(/second use/i && /aviation/i && /commercial/i);
```

Reply:
yeah I think that assertion is being a bit weird (or I'm doing something weird!) beause this passes for me

![image](https://user-images.githubusercontent.com/32230328/125828736-ddb19c35-4ecf-455f-9b1f-a3de6e2c426f.png)

does it only assert the last bit?

Reply:
Great catch!  Have replaced it with regex that works 👍. Thank you